storybook/src/components/MegaMenu/MegaMenu.stories.tsx

```diff
@@ -16,7 +16,9 @@ const meta = {
   decorators: [
     (Story) => (
       <Screen>
-        <Story />
+        <Grid paddingVertical="medium">
```

Comment:
Hmm, tricky. I’ve reverted the change to the Mega Menu stories. This required adding an extra Grid Cell to the Home Page example, but that’s not how we want it.

Reply:
Ah did not even check the homepage example. I must confess I'm not sure how you want to structure the a page now, it might help to know. Does this come close?

```jsx

<Screen>
  <Header {...props}>
    <MegaMenu>
       <Grid>
          <MenuItems />
       </Grid>
    </MegaMenu>
  </Header>
  <Grid>
    <Content />
  </Grid>
  <Footer>
    <FooterMenuItems />
  </Footer>
</Screen>

```

The polymorphic grid is going to throw (developer) people off, a Grid should not nest another Grid right?

Reply:
This is the reason, you are going to get misaligned grids:

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/b55acb0f-adf5-4ca3-84c2-73d40eae9f41">

Reply:
Fixed after the latest commits 👍 

The only issue is the wrapping `<nav>` element but this is hard.